setup.cfg

```diff
@@ -1,11 +1,12 @@
 [metadata]
 name = oneview-redfish-toolkit
-summary = OneView Redfish Toolkit
+summary = 'This toolkit provides a REST service to answer DMTF\'s Redfish compliant requests by querying HPE OneView.'
 description-file =
     README.md
-author = OneView Redfish Toolkit
-author-email = redfish-toolkit-l@lsd.ufcg.edu.br
+author = Hewlett Packard Enterprise Development LP
+author-email = oneview-pythonsdk@hpe.com
```

Comment:
Just suggestion, maybe the email could refer to "redfish-toolkit", something like oneview-redfish@hpe.com. Maybe after merge this PR we can do that, for example.